### PR TITLE
Print stats while running

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review.

### CODE REVIEW

**Comment by jonmv** [comment]
Requires https://github.com/vespa-engine/vespa/pull/18265

File: tests/performance/programmatic_feed_client/programmatic_feed_client.rb [hunk 1]
```diff
@@ -151,9 +149,9 @@ def run_benchmark_program(container_node, main_class, label, size)
     java_cmd =
       "java #{perfmap_agent_jvmarg} -cp #{java_client_src_root}/target/java-feed-client-1.0.jar " +
         "-Dvespa.test.feed.route=#{DUMMY_ROUTE} " +
-        "-Dvespa.test.feed.documents=#{(DOCUMENTS / (size / 10) ** (1.0 / 3)).to_i} " +
+        "-Dvespa.test.feed.documents=#{(DOCUMENTS} " +
         "-Dvespa.test.feed.document-text='#{generate_text(size)}' " +
-        "-Dvespa.test.feed.connections=8 " +
+        "-Dvespa.test.feed.connections=16 " +
```
Comment (bjorncs) [inline tests/performance/programmatic_feed_client/programmatic_feed_client.rb:154]: Connections is hardcoded here

**Review by bjorncs — CHANGES_REQUESTED** [review]

File: tests/performance/programmatic_feed_client/programmatic_feed_client.rb (hunk 1 above)
Comment (jonmv) [reply tests/performance/programmatic_feed_client/programmatic_feed_client.rb:154]: Thanks :>

Comment (jonmv) [reply tests/performance/programmatic_feed_client/programmatic_feed_client.rb:154]:
```suggestion
        "-Dvespa.test.feed.connections=#{connections} " +
```

File: tests/performance/programmatic_feed_client/programmatic_feed_client.rb [hunk 2]
```diff
@@ -151,9 +149,9 @@ def run_benchmark_program(container_node, main_class, label, size)
     java_cmd =
       "java #{perfmap_agent_jvmarg} -cp #{java_client_src_root}/target/java-feed-client-1.0.jar " +
         "-Dvespa.test.feed.route=#{DUMMY_ROUTE} " +
-        "-Dvespa.test.feed.documents=#{(DOCUMENTS / (size / 10) ** (1.0 / 3)).to_i} " +
+        "-Dvespa.test.feed.documents=#{(DOCUMENTS} " +
```
Comment (jonmv) [inline tests/performance/programmatic_feed_client/programmatic_feed_client.rb:152]:
```suggestion
        "-Dvespa.test.feed.documents=#{DOCUMENTS} " +
```

**Review by bjorncs — APPROVED** [review]

**Review by bjorncs — APPROVED** [review]

**Pull request merged**